Microsoft.Toolkit.Uwp.UI.Controls.Layout/Microsoft.Toolkit.Uwp.UI.Controls.Layout.csproj

```diff
@@ -15,7 +15,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.UI.Xaml" Version="2.3.200213001" />
+    <PackageReference Include="Microsoft.UI.Xaml" Version="2.4.0" />
```

Comment:
Same with the version here (2.4.2). Actually, it would be better to have this version on the Directory.Build.props, so we don't accidentally forget to update it at one place.